frontend/components/wallet/WalletDepositWithdrawBlock.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+    <div class="WalletDepositWithDrawBlock">
+        <TextBlock v-if="isExplanationsShown" title="Moving funds">
+            The DAI funds can be moved freely between VAT and your wallet without any extra conversion. You only need
+            to be aware that each transfer incurs transaction fees. Also, if you do not have DAI funds to deposit yet,
+            there are several ways to obtain it:
+            <ul class="list-disc list-inside">
+                <li>By borrowing DAI against a collateral in the oasis.app</li>
+                <li>
+                    By purchasing it on a decentralized exchange like uniswap.org (correct DAI token address used on
+                    the “mainnet” network is
+                    <FormatAddress type="address" :value="tokenAddressDAI" shorten />)
+                </li>
+            </ul>
+        </TextBlock>
+        <div>
+            <RadioGroup
```

Comment:
Please put it all inputs into a `form` element, so it's semantically correct form that can be operated from the keyboard and submitted via `enter` key